tests/foreman/data/hammer_commands.json

```diff
@@ -7188,7 +7188,7 @@
     },
     {
       "description": "Manipulate discovered rules.",
-      "name": "discovery-rule",
+      "name": "discovery_rule",
```

Comment:
Should we include a comment here? as JSON does not accept comments, usual pattern is to include as a key.
```json
{
  "description": "Manipulate dis...",
  "name": "discovery_rule",
  "__comment_name": "The names uses _ because of BZ 123456",
}
```

then we need to ensure that values with `startswith('__') are never read`